docs/examples/005-viz-with-holoviz.ipynb

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
<p>A few suggestions:</p><ol><li>Re-title: "Conversion to GeoDataFrame for Visualization with Holoviz"</li><li>Authors: We did not have authors section in any other usage example. I'd recommend to be consistent in that way.</li><li>Rephrase the main paragraph: "This notebook showcases how to use UXarray's utility functions for getting its data types (i.e. <code>Grid</code>, <code>UxDataArray</code>) ready to be visualized with HoloViz tools. Specifically, outlined is the conversion to a <code>GeoDataFrame</code> via UXarray's <code>to_geodataframe(), </code>which allows visualization with packages from the HoloViz stack. The notebook showcases basic visualization examples using the <code>hvPlot</code> package after the conversion."</li><li>Please add a note section after the main paragraph and tell that we will have Uxarray's basic plot() function in the near future. Maybe something like: "Conversion to <code>GeoDataFrame</code> works great to use it for visualization afterwards; however, we will release UXarray's own basic plotting function that will support Holoviz tools as backend in the near future, which could make visualization workflows for unstructured grids even more straightforward (i.e. as convenient as calling <code>ux.Grid.plot(...)</code> )"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>